src/index.js

```diff
@@ -62,6 +53,17 @@ toDoItems.forEach((item) => {
   item.addEventListener('mouseover', (event) => onMouseOver(event, item));
   item.addEventListener('mouseout', (event) => onMouseOut(event, item));
   item.addEventListener('dragstart', (event) => onDragStart(event, item));
-  item.addEventListener('drop', (event) => onDragEnd(event, item, toDoArr));
+  item.addEventListener('drop', (event) => onDragEnd(event, item, toDoItems, listWrap));
 });
 listWrap.addEventListener('dragover', (event) => onDragOver(event, listWrap));
+// ADD ITEM
+const addNewButton = document.querySelector('.add-new-btn');
+addNewButton.addEventListener('click', (event) => addNewItem(event, toDoArr, toDoItems, listWrap));
+// CLEAR ALL
+const clearAllBtn = document.querySelector('.clear-all');
+clearAllBtn.addEventListener('click', () => clearAll(listWrap));
```

Comment:
- [ ]  Or you target the check input of the item to set it clear, we do want just the one checked or completed to be removed.

Reply:
Thank you very much, I will fix it 😃